packages/typescript-estree/tests/ast-alignment/fixtures-to-test.ts

```diff
@@ -386,6 +386,11 @@ tester.addFixturePatternConfig('typescript/basics', {
      * This is intentional; babel is not checking types
      */
     'catch-clause-with-invalid-annotation',
+    /**
+     * [BABEL ERRORED, BUT TS-ESTREE DID NOT]
+     * SyntaxError: Unexpected token, expected ","
```

Comment:
babel/parser cannot parse override modifier for parameter properties, I'll report and fix it.

Reply:
https://github.com/babel/babel/issues/13427